server-phoenix/lib/helios/announcement.ex

```diff
@@ -20,11 +20,11 @@ defmodule Helios.Announcement do
     from q in query, where: q.location == ^location
   end
 
-  def happen_today(query, time_zone) do
-    {:ok, today} = DateTime.now(time_zone)
+  def happen_today(query) do
+    today = DateTime.new!(Date.utc_today(), ~T[00:00:00], "Etc/UTC")
+    # When DB is changed to Postgres (UTC -> Naive Time zone), the following code will be used:
```

Comment:
We switched the database to postgresql. We should probably test this out again

Reply:
It still works. There seems to be a problem with the weather poller though.

Reply:
I'm not sure about that. I have weather working locally.

Reply:
Oh ok. Must've been something with the env variables.